src/NumberGuessingGame.js

```diff
@@ -1,26 +1,26 @@
-import React, { Component, useState } from "react";
+import React from "react";
 import GuessControl from "./GuessControl";
 import GuessMessage from "./GuessMessage";
 import GameOver from "./GameOver";
 
 function NumberGuessingGame() {
-  const [numberToGuess, setnumberToGuess] = React.useState("");
-  const [numberOfGuesses, setnumberOfGuesses] = React.useState("");
+  const [numberToGuess, setnumberToGuess] = React.useState(getRandomNumber());
+  const [numberOfGuesses, setnumberOfGuesses] = React.useState(0);
   const [latestGuess, setlatestGuess] = React.useState("");
 
-  function handleGuess(event) {
-    setlatestGuess(parseInt(latestGuess));
-    setnumberOfGuesses(numberOfGuesses +1);
+  function handleGuess(guess) {
+    setlatestGuess(guess);
+    setnumberOfGuesses(parseInt(numberOfGuesses) + 1);
   }
   function handleReset() {
     const resetAll = {
       numberToGuess: getRandomNumber(),
       numberOfGuesses: 0,
       latestGuess: null,
     };
-    setnumberToGuess(resetAll);
-    setnumberOfGuesses(resetAll);
-    setlatestGuess(resetAll);
+    setnumberToGuess(resetAll.numberToGuess);
+    setnumberOfGuesses(resetAll.numberOfGuesses);
+    setlatestGuess(resetAll.latestGuess);
   }
   const isCorrectGuess = latestGuess === numberToGuess;
 
```

Comment:
I know this is not mentioned in the assignment but I would like for you to think of a design flaw in the way this current component is built. Look at the max attempts. Does this application do a good job at getting the user to abide by that max attempt?

Something to think about as you start building your own projects.